packages/icons/icons.json

```diff
@@ -3030,6 +3030,13 @@
         "group": "action",
         "codepoint": 62101
     },
+    {
+        "displayName": "Join: subtract",
+        "iconName": "subtract-join",
+        "tags": "circles, combine, connect, add, subtract, part, slice, intersection, venn diagram, intersect",
+        "group": "action",
+        "codepoint": 62354
+    },
```

Comment:
Can we add this entry to the end of the array? I'm a bit concerned on us properly incrementing the `codepoint` field here.